consent/types.go

```diff
@@ -27,6 +27,11 @@ import (
 	"github.com/ory/hydra/client"
 )
 
+const (
+	requestDeniedErrorName        = "Consent request denied"
```

Comment:
Errors names [should not start with a capital letter](https://github.com/golang/go/wiki/Errors) :)

Reply:
I like capitalization too much.

Fixed.